### DISCUSSION TITLE
Show Windows system tray menu on left click

### DISCUSSION THREAD

**Discussion by kurdin** [tauri-apps/tauri#7714] [General]
It is possible to show Windows system tray menu on left mouse click ? Right click works by default and it does not feel natural to click on system tray menu icon (right click) when to select menu item or open submenu (left click). 

Thank you for great framework!

**Comment by FabianLars** [comment]
> Right click works by default and it does not feel natural to click on system tray menu icon (right click)

That one is a bit surprising to me since right click for menu / left click to open app is how pretty much all the tray items on my pc work. I do agree that having options is (almost) always good :)

The problem with that is that as far as i know, the windows apis also are designed for right click and a semi-common workaround is to emit a right click event when a left click is detected.

Feel free to open a feature request though. Maybe the other maintainers know more than me.

**Comment by kurdin** [comment]
@FabianLars  Thank you for your quick response and for working on Tauri. I am enjoying development on Tauri platform. 

Maybe you right and for Windows' users its normal to right click to see a taskbar menu and left click to open the app associated with the menu. I am working on the app where user could quickly click on the dynamically created menu items to do different actions and I was thinking that it would be more convenient to use left click on Windows taskbar icon to access the menu. 

Tauri has already left click detect event for SystemTray, not sure why it was not added for Windows as option to show the task bar menu when it was implemented `menu_on_left_click` for a Mac. Seems like its was not that simple to add with Windows APIs. 

Thanks again!

**Reply by kurdin** [reply]
Created feature request 
https://github.com/tauri-apps/tauri/issues/7719
